src/Elastic.Markdown/Slices/Index.cshtml

```diff
@@ -16,7 +16,8 @@
 		StaticUrlPathPrefix = Model.StaticUrlPathPrefix,
 		GithubEditUrl = Model.GithubEditUrl,
 		AllowIndexing = Model.AllowIndexing,
-		Features = Model.Features
+		Features = Model.Features,
+		GetContentHashFunc = Model.GetContentHashFunc
```

Comment:
This can just be the `StaticFileContentHashProvider` instance

Reply:
https://github.com/elastic/docs-builder/pull/758/commits/d3e876fb0c3b26d8271a6bb61a41162dadb62bb7